.github/workflows/linter.yml

```diff
@@ -41,4 +41,6 @@ jobs:
       - name: Run mypy
         run: |
           pip install mypy
-          mypy .
+          pip install -r python/requirements.txt
+          mypy --config-file=.github/linters/mypy.ini run_ingestion/ python/ingestion/
```

Comment:
Any reason for type-checking these in the same line? Could you do this?
```
mypy run_ingestion/
mypy run_gcs_to_bq/
mypy python/
```

Reply:
Addressed in a comment above.